modules/channels.js

```diff
@@ -0,0 +1,32 @@
+var Channels = `
+                        <div class="image_holder" onclick="openImage('gallery/19.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/19.jpg">
+                            </div>
+                        </div>
+                        <div class="image_holder" onclick="openImage('gallery/20.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/20.jpg">
+                            </div>
+                        </div>
+                        <div class="image_holder" onclick="openImage('gallery/21.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/21.jpg">
+                            </div>
+                        </div>
+                        <div class="image_holder" onclick="openImage('gallery/22.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/22.jpg">
+                            </div>
+                        </div>
+                        <div class="image_holder" onclick="openImage('gallery/23.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/23.jpg">
+                            </div>
+                        </div>
+                        <div class="image_holder" onclick="openImage('gallery/18.jpg')">
+                            <div class="image_info">
+                                <img src="gallery/18.jpg">
+                            </div>
+                        </div>`;
+export default Channels;
```

Comment:
Add an empty line to the end of your files.

Reply:
okay sir